docs/docs/code/python/README.md

```diff
@@ -0,0 +1,279 @@
+# Writing Python in Code Steps
+
+**Anything you can do in Python can be done in a Pipedream Workflow**. This includes using any of the [350,000+ PyPi packages available](https://pypi.org/) in your Python powered workflows. 
+
+Pipedream supports [Python v{{$site.themeConfig.PYTHON_VERSION}}](https://www.python.org) in workflows.
+
+::: warning
+Python steps are available in a limited alpha release.
+
+You can still run arbitrary python code, including [sharing data between steps](/code/python/#sharing-data-between-steps) as well as [accessing environment variables](/code/python/#using-environment-variables).
+
+However, features available in [Node.js steps](/code/nodejs) like `$.respond`, `$.end`, and `$.auth` are not yet available in bash. If you have any questions please [contact support](https://pipedream.com/support).
+:::
+
+
+## Adding a Python code step
+
+1. Click the + icon to add a new step
+2. Click "Custom Code"
+3. In the new step, select the Python language runtime in language dropdown
+
+## Logging & debugging
+
+You can use `print` at any time in a Python code step to view information as the script is running.
+
+The output for the `print` logs will appear in the `Results` section just beneath the code editor.
+
+<div>
+<img alt="Python print log output in the results" src="./images/print-logs.png">
+</div>
+
+## Using 3rd party packages
+
+You can use any packages from [PyPi](https://pypi.org) in your Pipedream workflows. This includes popular choices such as:
+
+* `requests` for making HTTP requests
+* `sqlalchemy` for retrieving or inserting data in a SQL database
+* `pandas` for working with complex datasets
+
+To use a PyPi package, just include it in your step's code:
+
+`import requests`
+
+And that's it.
+
+No need to update a `requirements.txt` or specify elsewhere in your workflow of which packages you need. Pipedream will automatically install the dependency for you.
+
+## Making an HTTP request
+
+We recommend using the popular `requests` HTTP client package available in Python to send HTTP requests.
+
+No need to run `pip install`, just `import requests` at the top of your step's code and it's available for your code to use.
+
+### Making a GET request
+
+GET requests typically are for retrieving data from an API. Below is an example d
+
+```python
+import requests
+
+url = 'https://swapi.dev/api/people/1'
+
+r = requests.get(url)
+
+# to see the response in your Pipedream step logs:
+print(r.text)
+
+# to see the response status code in your Pipedream step logs:
+print(r.status)
+```
+
+### Making a POST request
+
+```python
+import requests
+
+# This a POST request to this URL will echo back whatever data we send to it
+url = 'https://postman-echo.com/post'
+
+data = {"name": "Bulbasaur"}
+
+r = requests.post(url, data)
+
+# to see the response in your Pipedream step logs:
+print(r.text)
```

Comment:
Most people will probably want to return the data for use in future steps. Maybe put the sharing data between steps section above this to introduce that concept, and then use the pipedream Python package to export the response data in this example.

Reply:
Yea good call, I was trying to introduce the full suite of HTTP verbs, but we should never condone bad security practices even if it's for a partial example.

Should never assume a person will read the docs in full, most people including me like to take snippets and move on to solve their problem.